### ISSUE TITLE
SystemVerilog Support

### ISSUE THREAD

**Issue by digee-bytes** [SpinalHDL/SpinalDoc-RTD#264]
Just like the VHDL or Verilog Generation, does SpinalHDL generate SystemVerilog codes and needs some basic info about using the verification environment in SpinalHDL?

**Comment by Dolu1990** [comment]
Hi,
The SpinalHDL system verilog backend is mostly the same than the verilog one,  with very minor differences.

Are you thinking about things as System verilog interface ?

**Comment by digee-bytes** [comment]
Yes, features like interfaces and support for UVM and complete verification flow such things

**Comment by Dolu1990** [comment]
ahhh so, SpinalHDL doesn't generate system verilog interface, the difference in generation is mostly in how assertion are emited

**Comment by mrcmry** [comment]
@digee-bytes based on the discussion, I closed this old issue. Feel free to reopen it if needed.
